x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/CsvTests.java

```diff
@@ -344,6 +344,10 @@ public final void test() throws Throwable {
                 "CSV tests cannot currently handle multi_match function that depends on Lucene",
                 testCase.requiredCapabilities.contains(EsqlCapabilities.Cap.MULTI_MATCH_FUNCTION.capabilityName())
             );
+            assumeFalse(
+                "CSV tests cannot currently handle CHUNK function",
```

Comment:
Why is this needed? Are we using something specific from Lucene on the chunker that will make this not to work?